spec/models/gallery_spec.rb

```diff
@@ -1,5 +1,24 @@
 # frozen_string_literal: true
 RSpec.describe Gallery do
+  before do
```

Comment:
let's do some validation tests by adding a context where the api_response is one with a URL without type image.
(expected to not be valid)